src/analysis/retail/warlock/affliction/constants.ts

```diff
@@ -30,3 +30,11 @@ export const getDotDurations = (combatant: Combatant): Record<number, number> =>
         ]),
       )
     : defaultDurations;
+
+// Shadow Embrace modifiers for Drain Soul / No Drain Soul
+export const shadowDefaultEmbraceModifier = 0.04;
+export const shadowDrainSoulEmbraceModifier = 0.02;
```

Comment:
Global constants should be `MACRO_CASE` like you've done with `SL_DAMAGE_BONUS`

Reply:
Implemented. So when should I use macro_case instead of lower case on the first word?

Reply:
Usually when declaring global variables. such as talent multipliers. Can also be used to replace magic numbers used for a specific analyzer to make it more clear what's going on.